package.json

```diff
@@ -26,21 +26,21 @@
   },
   "license": "MIT",
   "dependencies": {
-    "reflect-metadata": "0.1.9"
+    "reflect-metadata": "^0.1.12"
   },
   "peerDependencies": {
-    "graphql": "0.11.7"
+    "graphql": "0.13.2"
   },
   "devDependencies": {
-    "graphql": "0.11.7",
-    "mocha": "2.5.3",
-    "assert": "1.4.1",
-    "rimraf": "2.5.2",
-    "tslint": "5.1.0",
-    "typescript": "2.3.4",
-    "@types/graphql": "0.11.5",
-    "@types/mocha": "2.2.39",
-    "@types/node": "7.0.5"
+    "@types/graphql": "^0.13.1",
```

Comment:
We fix the version here to avoid problems with unintended updated

Reply:
I guess then it should be fixed to `0.13.1`